Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -2214,14 +2215,14 @@ std::pair<FixedContainerEdges, WeakElementEdges> LocalFrameView::fixedContainerE
         return side;
     };
 
-    auto lengthOnSide = [](BoxSide side, const LayoutRect& rect) -> LayoutUnit {
+    auto lengthOnSide = []<typename T>(BoxSide side, const T& rectOrSize) -> LayoutUnit {
```

Comment:
I think we prefer the explicit template argument list when the type `T` is used in the lambda body (say `T::foo`), but since that's not the case you may want to go with `auto`?


```suggestion
    auto lengthOnSide = [](BoxSide side, const auto& rectOrSize) -> LayoutUnit {
```

Reply:
Good idea, changed to use `auto`

Thanks for the review!